### DISCUSSION TITLE
Using image-weights in train.py

### DISCUSSION THREAD

**Discussion by PushpakBhoge512** [ultralytics/yolov5#9779] [Q&A] [answered]
Does anyone know how the train-weights feature works in train.py
do I need to define image weights in csv or how is it?

**Comment by glenn-jocher** [comment, marked as answer]
@PushpakBhoge512 --image-weights argument in train.py automatically loads images containing classes that are underperforming to overrepresent them during training. Nothing is required.

**Reply by PushpakBhoge512** [reply]
oh ok I actually wanted to specify per image weights (manually defined) to give some of the images more priority than other
The reason why I am doing this is that some of the images ~30% were incorrectly annotated so I wanted the model to relearn those

so I am guessing that the yolov5 does not support this out of the box, will need to modify the code for this right?

**Reply by glenn-jocher** [reply]
@PushpakBhoge512 yes you'd need to customize the dataloader sampler for this. All the dataloaders are in utils/dataloaders.py

**Reply by PushpakBhoge512** [reply]
Thank you @glenn-jocher will do!
